app/src/main/java/com/example/cm/data/models/MeetupNotification.java

```diff
@@ -1,34 +1,40 @@
 package com.example.cm.data.models;
 
-import androidx.annotation.NonNull;
-
-import java.util.Date;
-
 public class MeetupNotification extends Notification{
 
     private String meetupId;
     private String location;
     private String meetupAt;
+    private NotificationType type;
 
     public MeetupNotification(NotificationType type) {
         super();
         this.type = type;
         if(type == NotificationType.MEETUP_ACCEPTED || type == NotificationType.MEETUP_DECLINED){
-            state = NotificationState.NOTIFICATION_DECLINED;
+            state = NotificationState.NOTIFICATION_ANSWERED;
```

Comment:
why do we need NOTIFICATION_ANSWERED state?
think we are loosing information because of that